authappliance/lib/appliance.py

```diff
@@ -497,10 +497,7 @@ def halt(self, echo=False):
         """
         Shutdown OS
         """
-        p = Popen(["sudo", 'halt'],
-                  stdin=PIPE,
-                  stdout=PIPE,
-                  stderr=PIPE)
+        p = Popen(["sudo", 'halt'])
```

Comment:
I am not sure, if we need stdin and stdout. If not, we could also remove ``_output`` and ``_err`` further below.

Reply:
Done in c2ca84a